InaraResponder/InaraResponder.cs

```diff
@@ -1140,7 +1140,8 @@ private void handleCommanderReputationEvent(CommanderReputationEvent @event)
                         { "majorfactionReputation", @event.alliance / 100 }
                     }
                 }
-            }));
+            };
+            InaraService.Instance.EnqueueAPIEvent(new InaraAPIEvent(@event.timestamp, "setCommanderReputationMajorFaction", eventData));
```

Comment:
I've learned that Inara typically doesn't like us to send null data. 
We may want to separate
```cs
                    new Dictionary<string, object>()
                    {
                        { "majorfactionName", "independent" },
                        { "majorfactionReputation", @event.independent / 100 }
                    }
```
and null test it prior to adding it.

Reply:
Thanks.